src/drawer.multicanvas.js

```diff
@@ -482,6 +522,36 @@ export default class MultiCanvas extends Drawer {
         entry.setFillStyles(this.params.waveColor, this.params.progressColor);
     }
 
+    /**
+     * Called when an entry intersection goes above or below threshold.
+     *
+     * @param {array} entries
+     * @private
+     */
+    onIntersectionChange(entries) {
+        entries.forEach(entry => {
+            let canvasEntry = this.getCanvasEntryByElement(entry.target);
```

Comment:
Same here.

Reply:
im actually mutating canvasEntry one line below that so const won't work here.